accepted/PSR-17-http-factory.md

```diff
@@ -51,6 +51,8 @@ interface RequestFactoryInterface
 }
 ```
 
+The created instance MUST return an empty body-stream in read and write mode.
```

Comment:
As we discussed elsewhere, the spec itself cannot be changed. We can only update the meta document with errata.

Reply:
At least as far as adding **MUST** statements go.

Perhaps it is acceptable to add **SHOULD** statements... @php-fig/secretaries ?

Reply:
Because the bylaws say so?

I think you should vote on an exemption here - the spec is considerably less useful without these amendments being visible where they should be. If an implementer has to dig portions of the spec out of the meta section (which isn't even in the same file) that's quite a serious problem.

If what you're saying is, this isn't an "amendment" and can't be considered "errata", then it sounds like we're back to the idea of deprecating PSR-17.

Given the fact that the current spec does *not* guarantee interoperable implementations, in my opinion, this *is* errata. The definition of errata in the bylaws is somewhat ambiguous.

As I've argued before, this isn't a "breaking change" in the sense that it's going back on something that was already specified - we aren't making changes versus the previous specification, we're adding something that was unspecified and is currently inconsistent among existing implementations.

I sincerely hope we can amend rather than putting the entire community through the painful process of replacing the entire standard with a new one?